charts/securitywiki/templates/hpa.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.hpa.enabled }}
+apiVersion: autoscaling/v1
+kind: HorizontalPodAutoscaler
```

Comment:
Given that the wiki appears to be used by approximately 3 users, we probably don't strictly need this, but it won't hurt either.

Reply:
probably should remove at some point.